README.md

```diff
@@ -13,6 +13,7 @@
 
   1. Configure token to pull from the github npm repository
   2. `yarn` to install automatic dependencies
+    - If new dependencies are added to the project, you may need to temporarily delete the line `"@scientist-softserv/webstore-component-library": "VERSION_HERE",` from the `package.json` file in order to run `yarn` and install the new dependencies.
```

Comment:
what problems did you run into with new dependencies and the library dependency?

Reply:
see above comment

Reply:
I mean what dependency specifically, because I haven't had this problem that I recall.